src/components/sendV2/form/form.js

```diff
@@ -0,0 +1,320 @@
+      };
+    }
+
+    this.setState({
+      fields: {
+        ...this.state.fields,
+        recipient: {
+          ...this.state.fields.recipient,
+          ...recipient,
+        },
+      },
+    });
+  }
+
+  onSelectedAccount(account) {
+    this.setState({
+      fields: {
+        ...this.state.fields,
+        recipient: {
+          ...this.state.fields.recipient,
+          ...account,
+          value: account.address,
+          selected: true,
+          error: '',
+          feedback: '',
+          showSuggestions: false,
+        },
+      },
+    });
+  }
+
+  validateAmountField(value) {
+    if (/([^\d.])/g.test(value)) return this.props.t('Please use only digits and dots');
+    if (/(\.)(.*\1){1}/g.test(value) || /\.$/.test(value)) return this.props.t('Invalid amount');
+    return false;
+  }
+
+  // eslint-disable-next-line max-statements
+  validateAmountAndReference(name, value) {
+    const { t } = this.props;
+    const messageMaxLength = 64;
+    let feedback = '';
+    let error = '';
+
+    // istanbul ignore else
+    if (name === 'amount') {
+      value = /^\./.test(value) ? `0${value}` : value;
+      error = this.validateAmountField(value);
+      feedback = error || feedback;
+    }
+
+    // istanbul ignore else
+    if (name === 'reference' && value.length > 0) {
+      const byteCount = encodeURI(value).split(/%..|./).length - 1;
+      error = byteCount > messageMaxLength;
+      feedback = error
+        ? t('{{length}} extra characters', { length: byteCount - messageMaxLength })
+        : t('{{length}} out of {{total}} characters left', {
+          length: messageMaxLength - value.length,
+          total: messageMaxLength,
+        });
+    }
+
+    this.setState(prevState => ({
+      fields: {
+        ...prevState.fields,
+        [name]: {
+          error: !!error,
+          value,
+          feedback,
+        },
+      },
+    }));
+  }
+
+  onAmountOrReferenceChange({ target }) {
+    clearTimeout(this.loaderTimeout);
+
+    this.setState({ isLoading: true });
+
+    this.loaderTimeout = setTimeout(() => {
+      this.setState({ isLoading: false });
+      this.validateAmountAndReference(target.name, target.value);
+    }, 300);
+
+    this.onInputChange({ target });
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  onGoNext() {
+    Piwik.trackingEvent('Send_Form', 'button', 'Next step');
+  }
+
+  // eslint-disable-next-line complexity
+  render() {
+    const { t, followedAccounts } = this.props;
+    const { fields } = this.state;
+    const messageMaxLength = 64;
+    const byteCount = encodeURI(fields.reference.value).split(/%..|./).length - 1;
+    const isBtnDisabled =
+      fields.recipient.error || fields.amount.error || fields.reference.error ||
+      fields.recipient.value === '' || fields.amount.value === '';
+
+    return (
+      <div className={`${styles.wrapper}`}>
+        <header className={styles.header}>
+          <h1>{this.props.t('Send LSK')}</h1>
```

Comment:
Since the `t` is being destructured on the start of the render functions, not need for `this.props`